lib/p5.play.js

```diff
@@ -2314,8 +2314,8 @@ function Sprite(pInst, _x, _y, _w, _h) {
       // when objects end up at very high velocities (as happens sometimes in
```

Comment:
Does this actually fix it?  Your example case (long diagonal box vs small ball travelling on an axis) points out a fundamentally wrong assumption in my approach (see a few lines above this) in that the centers of the colliding objects may pass each other on the velocity axis _before_ we would expect the objects to actually collide - meaning we may end up significantly oversampling before the collision and still missing the actual collision.  (I guess the reverse can happen too - you collide well before the centers overlap).

This may call for a more drastic rethinking of the tunneling prevention.  We should get your change in anyways, since it seems to improve things.